src/App.js

```diff
@@ -1,20 +1,37 @@
+// import React from 'react';
+// import './index.css';
+// import { BrowserRouter, Routes, Route } from 'react-router-dom';
+// import Header from './components/Header';
+// import Books from './components/Books';
+// import Categories from './components/Categories';
+
+// // eslint-disable-next-line react/prefer-stateless-function
+// const App = () => (
+//   <BrowserRouter>
+//     <Header />
+//     <Routes>
+//       <Route path="/" element={<Books />} />
+//       <Route path="/Categories" element={<Categories />} />
+//     </Routes>
+//   </BrowserRouter>
+
+// );
```

Comment:
Please be sure to remove your commented code before creating a PR. By removing them you will not only have cleaner code but your repo will be even more professional 😉

Reply:
ok